torchbenchmark/models/detectron2_maskrcnn/__init__.py

```diff
@@ -0,0 +1,83 @@
+import torch
+import os
+import itertools
+import random
+
+# TorchBench imports
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+# setup environment variable
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+DATA_DIR = os.path.join(CURRENT_DIR, ".data", "coco2017-minimal")
+if not 'DETECTRON2_DATASETS' in os.environ:
+    os.environ['DETECTRON2_DATASETS'] = DATA_DIR
+
+from detectron2.config import instantiate
+from detectron2 import model_zoo
+from detectron2.utils.collect_env import collect_env_info
+from detectron2.utils.logger import setup_logger
+from detectron2.utils.events import EventStorage
+
+torch.manual_seed(1337)
+random.seed(1337)
+torch.backends.cudnn.deterministic = False
+torch.backends.cudnn.benchmark = False
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.DETECTION
+
+    # This model doesn't support setting batch size for inference
+    def __init__(self, device=None, jit=False, train_bs=4, eval_bs=1):
+       super().__init__()
+       self.device = device
+       self.jit = jit
+
+       model_cfg = model_zoo.get_config("common/models/mask_rcnn_fpn.py").model
```

Comment:
does this read from some github repo?

Reply:
This reads from https://github.com/facebookresearch/detectron2/blob/main/configs/common/models/mask_rcnn_fpn.py.
The package is installed by this pip dependency: https://github.com/pytorch/benchmark/pull/556/files#diff-c42e6d96498656cf306797b2fbae3c2c7172eb1ec2305c0746c9a8cdb832db0fR1.